prov/rxm/src/rxm.h

```diff
@@ -140,6 +140,7 @@ extern struct fi_ops_atomic rxm_ops_atomic;
 extern size_t rxm_msg_tx_size;
 extern size_t rxm_msg_rx_size;
 extern size_t rxm_def_univ_size;
+extern size_t rxm_poll_freq_usec;
```

Comment:
Can we rename this variable to `rxm_cm_progress_interval`?

Reply:
Sure.